scripts/ratchet_coverage/install_cargo_llvm_cov.py

```
@@ -4,16 +4,19 @@
 # dependencies = ["plumbum", "typer"]
 # ///
 from plumbum.cmd import cargo
+from plumbum.commands.processes import ProcessExecutionError
 import typer
 
 
 def main() -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)